javascript/ql/src/Security/CWE-400/PrototypePollutionUtility.ql

```diff
@@ -420,12 +518,18 @@ string deriveExprName(DataFlow::Node node) {
  * In most cases this will result in an alert, the exception being the case where
  * `base` does not have a prototype at all.
  */
-predicate isPrototypePollutingAssignment(Node base, Node prop, Node rhs, EnumeratedPropName enum) {
+predicate isPrototypePollutingAssignment(Node base, Node prop, Node rhs, Node propNameSource) {
```

Comment:
The above qldoc still mentions a property `enum`, which has been renamed to `propNameSource`.